Validate bug readme (#891)
demisto/demisto-sdk: CHANGELOG.md

* Updated the **download** command to support mappers.
* Fixed an issue where the ***format*** command added a duplicate parameter.
* For partner supported content packs, added support for a list of emails.

> * Fixed an issue where the ***validate*** command failed when pack has no README file.

Reviewer: ```suggestion
* Removed validation of README files from the ***validate*** command.
```
